assets/store/User.js

```diff
@@ -104,6 +105,7 @@ export default class User extends Reactive {
       default_connection: data.default_connection || '',
       forced_connection: data.forced_connection || false,
       highlightKeywords: data.highlight_keywords || [],
+      ignoreStatuses: data.ignoreStatuses || [],
```

Comment:
We are not able to handle `ignoreStatuses` in the backend yet, so this won't work. You either have to update the Perl code and API spec or remove this.